Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -39,14 +39,18 @@ extension Effect where Failure == Never {
 }
 
 struct AnimationsState: Equatable {
-  var circleCenter = CGPoint.zero
+  var alert: AlertState<AnimationsAction>? = nil
+  var circleCenter = CGPoint(x: 50, y: 50)
   var circleColor = Color.white
   var isCircleScaled = false
 }
 
 enum AnimationsAction: Equatable {
+  case alertButtonTapped
```

Comment:
What do you think of changing the name to something a little more specific and re-alphabetizing?
```suggestion
  case resetConfirmationButtonTapped
```